smart-contracts/contracts/interfaces/IUnlock.sol

```diff
@@ -27,7 +27,8 @@ interface IUnlock {
   function createLock(
     uint _expirationDuration,
     uint _keyPrice,
-    uint _maxNumberOfKeys
+    uint _maxNumberOfKeys,
+    uint _cancelRefundPenaltyDenominator
```

Comment:
this could be removed _IF_ we decide to just set a default value in the lock constructor and provide a setter.

Reply:
Yes!

Reply:
Done